### DISCUSSION TITLE
Should we release 0.71.8?

### DISCUSSION THREAD

**Discussion by fortmarek** [reactwg/react-native-releases#65] [Patches]
## Released, please refer to #69

---

Current Release: [0.71.7](https://github.com/facebook/react-native/releases/tag/v0.71.7)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. 
#### Local commits to backport to main

1.

**Comment by kelset** [comment]
from @renchap:

> https://github.com/facebook/react-native/commit/a804c0f22b4b11b3d9632dc59a6da14f6c4325e3 to fix onChangeText firing on TextInput mount (fixing regression https://github.com/facebook/react-native/issues/36494)

**Comment by ste7en** [comment]
Hi, is it possible to include the CLI version bump from https://github.com/facebook/react-native/commit/c5a47abaf894235f71eea03e57d2f5287d80f3d4 to address an issue described here https://github.com/facebook/react-native/issues/37110 with version 10.2.0 ?

**Reply by kelset** [reply]
we can't bump CLI to 11.x in 0.71.
0.71 is on 10.x.

So to do what you are asking, we need the CLI team to backport https://github.com/react-native-community/cli/pull/1829 to the 10.x branch (https://github.com/react-native-community/cli/commits/10.x), and do a new release.

**Reply by kelset** [reply]
update: I've talked with the CLI folks, it sounds like cherry-picking the commit needed would be fairly complicated because it builds on top of some other important refactoring that were done in the v11 of the CLI. So we can't address it.

**Comment by kelset** [comment]
once merged, we need to pick this: https://github.com/facebook/react-native/pull/37204

**Comment by leotm** [comment]
could we pos bump [0.71-stable/sdks/.hermesversion](https://github.com/facebook/react-native/blob/0.71-stable/sdks/.hermesversion) `hermes-2023-03-07-RNv0.71.4-31fdcf738940875c9bacf251e149006cf515d763`
to address this Hermes crash https://github.com/facebook/hermes/issues/886 (so bump would include https://github.com/facebook/hermes/commit/f2f1a2e38e86632f24176d9e607c06507f900496 from `2023-03-23`)

**Reply by kelset** [reply]
@leotm sorry if it's a dumb question, but how do you know that https://github.com/facebook/hermes/commit/f2f1a2e38e86632f24176d9e607c06507f900496 fixes the reported problem? in that issue there's no mention of this commit/fix - also, that issue is about RN 0.70.

**Reply by leotm** [reply]
all good! for our [RN 0.71.6 upg](https://github.com/MetaMask/metamask-mobile/pull/6220) (wip) we [patched](https://github.com/MetaMask/metamask-mobile/pull/6220/commits/794e7c05e41e9ac52da67391b1ca5374579fc1d5) the lib, then later [updated](https://github.com/MetaMask/metamask-mobile/pull/6220/commits/5dc1949c5757f8a3e324ce004b24779cadb64ef0#diff-9a50ef936e92a26017fd739f42ee0a0e7f80aec39dafc0a13f27b2548ee5e51f) the proj to build from source with [custom](https://github.com/MetaMask/metamask-mobile/pull/6220/commits/8a5dce5433b0ca5fc5075e047b5d3e21c2ff2b56) Hermes build (0.71.6 tag + https://github.com/facebook/hermes/commit/f2f1a2e38e86632f24176d9e607c06507f900496 cherry-picked) and tested w/o the lib patch (gr8 success)
[removed](https://github.com/MetaMask/metamask-mobile/pull/6220/commits/8da441569ddbb283dca35f3e6b9b2997d9ee5c0f) it now and [re-tested](https://github.com/MetaMask/metamask-mobile/pull/6220#issuecomment-1540765257) to confirm

**Reply by kelset** [reply]
thanks for testing - we had to "rush" a bit this patch so we'll handle your request in the next one, I've already added a comment there about it.
